docs/xdr/features/collect/integrations/endpoint/sentinelone_deepvisibility.md

```diff
@@ -1,5 +1,5 @@
 uuid: 8c1bee36-d516-42f9-9b6f-a8e4dcac3d1d
-name: SentinelOne Deep Visibility
+name: SentinelOne Deep Visibility [DEPRECATED]
```

Comment:
The name is not synced with this intake-format repository

https://github.com/SEKOIA-IO/intake-formats/commit/990e739f7dc1f6ca759c686dbfa3bb7ad61f35e1#diff-24e16cf9f977119509f99a9de017e9f90b1dce102773faa718d77c35df5e977fR2